lib/src/ansi_code.dart

```diff
@@ -66,20 +69,31 @@ class AnsiCode {
   const AnsiCode._(this.name, this.type, this.code, this.reset);
 
   /// Represents the value escaped for use in terminal output.
-  String get escape => "\x1B[${code}m";
+  String get escape => "$_ansiEscapeLiteral[${code}m";
+
+  /// Represents the value as an unescaped literal suitable for scripts.
+  String get forScript => "$_ansiEscapeForScript[${code}m";
```

Comment:
escapeForScript?

Feel like we're overloading the forScript variable.

Reply:
Overloading the `forScript` argument? Yeah.

I had `escapeForScript` before. I guess I could go back there.

Thoughts, @matanlurey ?

Reply:
@alorenzen – PTAL